returnn/tf/layers/rec.py

```diff
@@ -4363,6 +4363,7 @@ def __init__(self, beam_size, keep_beams=False,
                input_type="prob",
                prob_scale=1.0, base_beam_score_scale=1.0, random_sample_scale=0.0,
                length_normalization=True,
+               length_normalization_alpha=1.0,
```

Comment:
Is this a good name? Maybe better `length_normalization_exp`, or `..._exponent`? I don't really like names like "alpha", "beta", "x", "y", "a", "b", which are all really arbitrary.